usaspending_api/awards/v2/lookups/elasticsearch_lookups.py

```diff
@@ -25,6 +25,72 @@
     "Last Date to Order": "ordering_period_end_date",
 }
 
+default_mapping = {
```

Comment:
is this a "default" that ever gets used directly, or a "base" that is only ever appended to?

Reply:
It only gets appended to

Reply:
Probably better to name it `base_mapping` for clarity. In general in this area, this could be shortened a lot by defining rather than modifying dependent dictionaries:
```
base_mapping = {"blah":"blah"}
contracts_mapping = {**base_mapping, **{"yada":"yada"}}
```

Reply:
Good thoughts. The proposed syntax was unavailable when the original code this is based on was written.

Reply:
sure